submissions/korotushak96/js-dom/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Also, I strongly recommend you to configure your IDE for adding an empty line at the end of each file.
It will take fewer efforts to review your code in the future.
@OleksiyRudenko also explained this feature in the first lecture for p2p course.

Reply:
May you recommend some extension for VS code?

Reply:
I think you can configure it from your IDE settings. To my mind, this operation does not need some extensions.

Reply:
ok. Thanks